### ISSUE TITLE
LineContinuationBear: Ignore backslashes in doctest strings automatically

### ISSUE THREAD

**Issue by Kiy4h** [coala/coala-bears#2805]
LineContinuationBear should ignore lines like [this](https://github.com/coala/coala/blob/master/coalib/bearlib/languages/documentation/DocumentationExtraction.py#L214) automatically:

```python
malformed_comment = MalformedComment(dedent("""\
```

**Comment by naveentvelu** [comment]
@jayvdb 
To maintain a `flag` variable which has default value `false`. When a triple quoted string is encountered we can set the `flag` to `true`. And on the second encounter we can unset the `flag`.
If the `flag` is **set** (i.e flag = true) we can Ignore the backslashes.

**Comment by li-boxuan** [comment]
@naveentvelu Are you still keen on solving this issue? Let me know if not, then I'll reassign to @mk1299

**Comment by naveentvelu** [comment]
Yeah I will work on this issue. But I am having typhoid for past two days. Once i am okay i will make a PR.If this issue is to be solved very urgently then you can assign to someone else.

**Comment by li-boxuan** [comment]
> Once i am okay i will make a PR.If this issue is to be solved very urgently then you can assign to someone else.

Nope, not urgent :) Good luck

**Comment by rajdeepbharati** [comment]
@naveentvelu are you still working on this?

**Comment by Apoorv01017** [comment]
can I work on this issue?
